src/modules/TextInput/TextInput.tsx

```diff
@@ -136,39 +137,35 @@ export const TextInput: React.FC<PropsWithChildren<
       >
         {label}
       </legend>
-      {label && (
-        <span ref={hiddenLabel} className={cx(styles.hiddenLabel)}>
+      <span ref={hiddenLabel} className={cx(styles.hiddenLabel)}>
```

Comment:
Same question as above: if the label is not necessarily gonna be passed, why render this element?

Reply:
label is always passed as "" if not specified. I just preferred not to keep conditions on render

Reply:
Yeah I understand that, that way you are forcing the element to always be rendered. But why? Is it for a similar reason you mentioned above, i.e. having consistent spacing? Contrary to the error label though, this label will not be potentially changing right?
This is a minor thing anyway 🙂